bootstrap.sh

```diff
@@ -0,0 +1,14 @@
+echo "Setting up docker-compose.yml file, if it doesn't exist already"
```

Comment:
I had to add a shebang for this to execute, like 
`#!/bin/bash`

Reply:
The latest commit removes bootstrap.sh and makes it a make target, as in `make bootstrap`